src/modules/dashboard/data/queries/colonies.ts

```diff
@@ -501,11 +501,9 @@ export const getColonyTokenBalance: Query<
       const {
         total: nonRewardsPotsTotal,
       } = await colonyClient.getNonRewardPotsTotal.call({ token });
-      return new BigNumber(
-        nonRewardsPotsTotal.add(rewardsPotTotal).toString(10),
-      );
+      return new BigNumber(nonRewardsPotsTotal.add(rewardsPotTotal));
```

Comment:
Ahhh I'm sorry, it seems I was mistaken about the BN constructor taking another BN 🙇 this is throwing an error now :(

Reply:
Shoooot I swear I tested it with this... But you're right, broken indeed!

I've dropped this commit.